codestyle/checkstyle.xml

```diff
@@ -60,7 +60,6 @@
             <property name="allowByTailComment" value="true"/>
             <property name="allowNonPrintableEscapes" value="true"/>
         </module>
-        <module name="AvoidStarImport"/>
```

Comment:
Do we really want to change the checkstyle definition with this PR? Also, this rule seems to be an easy one to meet...

Reply:
This change is only needed for two instances of `KeyFigureCardSequenceConstants.*`...